Files/Filesystem/Search/FolderSearch.cs

```diff
@@ -31,26 +34,55 @@ internal class FolderSearch
 
         private uint UsedMaxItemCount => MaxItemCount > 0 ? MaxItemCount : uint.MaxValue;
 
-        public async Task<ObservableCollection<ListedItem>> SearchAsync()
+        public EventHandler SearchTick;
+
+        public string AQSQuery
         {
-            var results = new ObservableCollection<ListedItem>();
+            get
+            {
+                // if the query starts with a $, assume the query is in aqs format, otherwise assume the user is searching for the file name
+                if (Query is not null && Query.StartsWith("$"))
+                {
+                    return Query.Substring(1);
+                }
+                else
+                {
+                    return $"filename:\"{Query}\"";
```

Comment:
AQS query syntax is localized "filename:XXX" only works is system language is English. I think you have to use `System.Filename:XXX`.